### ISSUE TITLE
"Cannot read property 'startContainer' of null" with b element

### ISSUE THREAD

**Issue by blocknotes** [Alex-D/Trumbowyg#1233]
_"Cannot read property 'startContainer' of null"_ JavaScript error with the following initial content: `<p><b>Test</b></p>`

### Description
Using Trumbowyg 2.25.0 and setting `<p><b>Test</b></p>` as initial content for the editor I get the JS error _"Cannot read property 'startContainer' of null"_ and the toolbar is not available.

### How to reproduce?
https://jsfiddle.net/dLvfeaob/1/

```html
<div id="editor">
  <p><b>Test</b></p>
</div>
```

```js
$(document).ready(function() {
  $('#editor').trumbowyg();
});
```

**Comment by Alex-D** [comment]
Thank you for the report!
I've fixed it in 2.25.1 :)

**Comment by blocknotes** [comment]
wow 🚀 thanks @Alex-D
